presto-kafka07/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -76,7 +76,17 @@
     /**
      * Fetch size
      */
-    private int fetchSize = 10 * 1024 * 1024;
+    private DataSize fetchSize = new DataSize(10, Unit.MEGABYTE);
+
+    /**
+     * Default Query Interval
+     */
+    private Duration defaultQueryInterval = Duration.valueOf("10m");
+
+    /**
+     * Hard limit on
```

Comment:
Also, can you elaborate more about what this variable means? I have to read through the logic code using this variable to understand it.